java-plugin/src/test/files/CookieWithoutExpirationCheck.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+
+class TestClass {// Noncompliant {{Avoid not setting MaxAge}}
```

Comment:
why message presnet on class and not on each line of code with a non compliance ? I think this is that we should have.